src/js_native_api_v8.cc

```diff
@@ -2589,8 +2589,13 @@ napi_status napi_create_arraybuffer(napi_env env,
   CHECK_ARG(env, result);
 
   v8::Isolate* isolate = env->isolate;
+  std::unique_ptr<v8::BackingStore> backing =
+      v8::ArrayBuffer::NewBackingStore(isolate, byte_length);
   v8::Local<v8::ArrayBuffer> buffer =
-      v8::ArrayBuffer::New(isolate, byte_length);
+      v8::ArrayBuffer::New(isolate, std::move(backing));
+  // TODO(thangktran): drop this check when V8 is pumped to 8.0 .
+  if (!buffer->IsExternal())
+    buffer->Externalize(buffer->GetBackingStore());
```

Comment:
This wasn’t externalized previously, is there a reason to do so now?

(Also, this is one of the cases where `ArrayBuffer::New()` is not going to be deprecated)

Reply:
fixed.